### PR TITLE
Added the corresponding data for Assigment1

### PR DESCRIPTION
**Pull request by ErHulio**

### CODE REVIEW

File: Assignment4/ErHulio-z170088/task06.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, VCARD.Given, Literal("Jane")))
+g.add((ns.JaneSmith, VCARD.Family, Literal("Smith")))
+
+# %% [markdown]
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# %%
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.worksAt, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/ErHulio-z170088/task06.py:60]: vcard does not definde the property worksAt. You have to search for another vocabulary or define your own property

File: Assignment4/ErHulio-z170088/task07.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  print(s)
+
+print()
+
+q2 = prepareQuery('''
+  SELECT 
+    ?Subject ?Subject2
+  WHERE {
+    ?Subject rdf:type ns:Person.
+    ?Subject2 rdf:type ns:Researcher.
```
Comment (dgarijo) [inline Assignment4/ErHulio-z170088/task07.py:62]: This is printing all the entities which are of type person and researcher. Not those from all subclasses of person

File: Assignment4/ErHulio-z170088/task07.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+# 
+
+# %%
+for s, p, o in g.triples((None, RDF.type, ns.Person)):
+  for w, x, y in g.triples((s, None, None)):
+    print(s, x, y)
+
+for s, p, o in g.triples((None, RDF.type, ns.Researcher)):
+  for w, x, y in g.triples((s, None, None)):
+    print(s, x, y)
```
Comment (dgarijo) [inline Assignment4/ErHulio-z170088/task07.py:83]: There are more sublcasses of person. You have to do this method without listing them manually

File: Assignment4/ErHulio-z170088/task07.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+q3 = prepareQuery('''
+  SELECT 
+    ?Subject ?Predicate ?Object ?Subject2 ?Predicate2 ?Object2
+  WHERE {
+    ?Subject rdf:type ns:Person.
+    ?Subject ?Predicate ?Object.
+
+    ?Subject2 rdf:type ns:Researcher.
+    ?Subject2 ?Predicate2 ?Object2.
```
Comment (dgarijo) [inline Assignment4/ErHulio-z170088/task07.py:95]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
There are incorrect things. I also believe the deadline for assignment 4 has passed.

**Comment by ocorcho** [comment]
Closing as comments have not been taken into account for a long period.

**Pull request closed without merging**